### ISSUE TITLE
link to external listing of scikits is no longer working

### ISSUE THREAD

**Issue by grlee77** [scipy/scipy.org#382]
On SciPy's [topical software page](https://scipy.org/topical-software.html), there is a link to a "List of SciKits" at the right (see image below):

That link points to: http://scikits.appspot.com/scikits, but I get a "Server Error" when following the link

The top-level http://scikits.appspot.com page loads, but the "source" link in the footer of that page points to (https://bitbucket.org/janto/scikits_index/), which no longer seems to exist.

![scikits](https://user-images.githubusercontent.com/6528957/106612001-96f1cb00-6536-11eb-862a-70d01a6b8fbd.png)

**Comment by rgommers** [comment]
Maybe we should just get rid of that content? Or move it into a page on this repo. The scikits.appspot.com hasn't really made sense for at least 10 years.

Cc @stefanv, @jarrodmillman who may remember the origins of that site.

**Comment by grlee77** [comment]
Yeah, I think at this point there a lot of them, so it doesn't seem practical to maintain a comprehensive list. Maybe link to something like: https://pypi.org/search/?q=scikit&o=&c=Intended+Audience+%3A%3A+Science%2FResearch

If we are going to keep a topical page, it seems odd for it not to mention popular packages like scikit-learn or scikit-image, though. 

Searching by tags/keywords on places like PyPI and GitHub is a reasonable way for users to find more comprehensive package lists.

**Comment by stefanv** [comment]
Yes, a student built that for me a long, long time ago.  The code is now at https://github.com/stefanv/scikits_index/  It was running until a short while ago and gets a fair number of hits still, so I'll at least see if it's a quick fix.

I'm not sure how relevant it is any longer, at least not without some work.  It used to make sense because our scikits used to be distributed across so many different places.

**Comment by stefanv** [comment]
```
<Fault -32500: "RuntimeError: PyPI's XMLRPC API has been temporarily disabled due to unmanageable load and will be deprecated in the near future. See https://status.python.org/ for more information.">
```
